pkg/projectfile/projectfile.go

```diff
@@ -972,9 +975,51 @@ func createCustom(params *CreateParams, lang language.Language) (*Project, error
 		return nil, err
 	}
 
+	if params.Cache != "" {
+		createErr := createCacheFile(params.Directory, params.Cache)
+		if createErr != nil {
+			return nil, errs.Wrap(createErr, "Could not create cache file")
+		}
+	}
+
 	return Parse(params.path)
 }
 
+func createCacheFile(filePath, cachePath string) error {
+	user, err := user.Current()
+	if err != nil {
+		return errs.Wrap(err, "Could not get current user")
+	}
+
+	data := map[string]interface{}{
+		"Cache": cachePath,
+	}
+
+	tplName := "activestate.yaml.cache.tpl"
+	tplContents, err := assets.ReadFileBytes(tplName)
+	if err != nil {
+		return errs.Wrap(err, "Could not read asset")
+	}
+
+	fileContents, err := strutils.ParseTemplate(string(tplContents), data)
+	if err != nil {
+		return errs.Wrap(err, "Could not parse %s", tplName)
+	}
+
+	// Username on Windows is  DOMAIN\username, we only want the username
+	// rather than trimming the domain, we just take the last part of the
+	// home directory's path
```

Comment:
Some users have different usernames than their home directory (not just on Windows, but on any platform). Some usernames may also have spaces in them. Is this going to cause any trouble with the logic here?

Reply:
Good point. Updated the username handling for windows and also ensure that the username has no spaces on any platforms.